client/src/constants/forms.js

```diff
@@ -1,27 +1,11 @@
+/* eslint-disable */
```

Comment:
Does ESLint need to be disabled for the whole file?

Reply:
Was a straggler. Fixed.